components/benefit_sponsors/app/models/benefit_sponsors/locations/address.rb

```diff
@@ -284,6 +284,10 @@ def office_kinds
         OFFICE_KINDS
       end
 
+      def display_address
```

Comment:
given that this is really a view helper, I think this probably shouldn't go on the model?